### PR TITLE
Group 404 checkboxes and shorten their labels

### LINKED ISSUE #2250: 404 Redirection
The checkbox labels are hard to read (length) and cannot be differentiated easily (similarity):

bildschirmfoto 2014-11-10 um 13 38 38

### PR DESCRIPTION
**Pull request by nitriques**
Trying to associate the first 2 checkbox with their respective textbox
while having a unrelated thrid checkbox is confusing. Also, all labels
were are hard to read (length) and cannot be differentiated easily
(similarity)

Fixes #2250

### CODE REVIEW

**Comment by nitriques** [comment]
Without fieldset:
screen shot 2017-06-06 at 10 00 33 am

**Comment by nitriques** [comment]
With a fieldset:
screen shot 2017-06-06 at 10 05 16 am

With this perspective, I think the wording must be adjusted. Conditions could become Parameters ?

File: symphony/content/content.blueprintsdatasources.php [hunk 1]
```diff
@@ -324,22 +324,26 @@ public function __form()
 
         $fieldset->appendChild($group);
 
+        $this->Form->appendChild($fieldset);
+
+        $fieldset = new XMLElement('fieldset');
+        $this->setContext($fieldset, array('sections', 'system'));
+        $fieldset->appendChild(new XMLElement('legend', __('Page not found')));
+        $p = new XMLElement('p', __('Redirect to 404 page by enabling those criteria.'));
```
Comment (nilshoerrmann) [inline symphony/content/content.blueprintsdatasources.php]: This copy reads a bit strange. Maybe @brendo has an idea how to improve it.

**Comment by nilshoerrmann** [comment]
Maybe the legends could read "Execution Rules" and "Redirect Rules". The second rule could then be changed to:

> # Redirect Rules
> Enabling these conditions will throw a 404 page not found error.
>
> - [ ] required parameter missing
> - [ ] forbidden parameter present
> - [ ] no results

**Comment by nitriques** [comment]
I am not found of the "Redirection", since it is not a 30x status, but a real 404. But that's the engineer speaking ;)

File: symphony/content/content.blueprintsdatasources.php (hunk 1 above)
Comment (nitriques) [inline symphony/content/content.blueprintsdatasources.php]: Yes, yes!

**Comment by nilshoerrmann** [comment]
In this case "Error Rules" might be a engineer-friendly alternative ;)

**Comment by nitriques** [comment]
I like it!
screen shot 2017-06-06 at 10 05 16 am

**Comment by nitriques** [comment]
I would be good to merge as-is, if nobody is opposed.

**Comment by brendo** [comment]
What about `Execution Conditions` and `Error Conditions`? Otherwise fine

**Comment by nitriques** [comment]
Yes `conditions` is better (Since they are not rules *per se*)
@nilshoerrmann ?

**Comment by nitriques** [comment]
@nilshoerrmann if you could approve the PR that would be neat! Thanks!

**Comment by nitriques** [comment]
> I can’t approve it, though, I guess because it's already merged?

Yup ;)

BTW, the review thing is really great 👍

**Pull request merged**